oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java

```diff
@@ -67,9 +67,13 @@ public byte[] getBytes(String fieldName) {
     public static class StreamToStorage implements Convert2Storage<StreamWrite> {
         private final MetadataRegistry.Schema schema;
         private final StreamWrite streamWrite;
+        private final String timestampColumn;
```

Comment:
I suppose this field can be incorporated into the `MetadataRegistry.Schema` during the model parse, so we don't need this field here.

Reply:
make sense 👍